### DISCUSSION TITLE
Outbox for PMs

### DISCUSSION THREAD

**Discussion by julkasalsa** [RetroAchievements/RAWeb#693] [Ideas]
Allow Outbox/Sent box for PMs, and fix text aligment and formatting.

**Comment by Jamiras** [comment]
Partially implemented by #912, which gives users the ability to see emails they sent as long as the recipient hasn't deleted them.

**Reply by Jamiras** [reply]
PMs are now threaded (via #2062). You can see any chain you've participated in as long as you haven't marked it as deleted.
